src/admin-views/widgets/list.php

```diff
@@ -23,6 +23,26 @@
 }
 
 foreach ( $admin_fields as $field_id => $field ) {
+	// Handle a section start. May contain a section title.
+	// Using stripos() to allow for multiple occurrences ("section_start_1", "section_start_2" etc).
+	if ( false !== stripos( $field_id, 'section_start' ) ) {
+	?>
+		<div class="tribe-events-widget-admin-form__input-section">
+		<?php if ( ! empty( $field['title'] ) ) { ?>
+			<h4><?php echo esc_html( $field['title'] ); ?></h4>
+		<?php
+		}
+
+		continue;
+	}
+
+	// Handle a section end.
+	if ( false !== stripos( $field_id, 'section_end' ) ) {
```

Comment:
Same here: since we're checking for fields IDs that start with `section_end`, I would use `0 === stripos` here.